### PR TITLE
Bump cpython artifacts

### PR DESCRIPTION
**Pull request by woodruffw**
See https://github.com/woodruffw/cpython-release-tracker/pull/7.

### CODE REVIEW

**Comment by kommendorkapten** [comment]
The change looks ok in that it references the merge commit, give me a ping when the issues with verification are sorted out 👍

**Comment by jku** [comment]
`Certificate's OIDCIssuer does not match (got 'https://github.com/login/oauth', expected 'https://accounts.google.com')`

does the cpython release system allow any kind of signature or how does this happen?

**Comment by di** [comment]
> does the cpython release system allow any kind of signature or how does this happen?

It allows any issuer, and this looks like it's due to https://github.com/sigstore/sigstore-python/issues/1204

**Comment by woodruffw** [comment]
Yep, that was the problem -- sorry, I forgot to follow up on this earlier. Now that that's fixed I should be able to rebuild that table and this will pass.

**Comment by di** [comment]
I think you still need https://github.com/woodruffw/cpython-release-tracker/pull/8 as well, no?

**Comment by woodruffw** [comment]
> I think you still need [woodruffw/cpython-release-tracker#8](https://github.com/woodruffw/cpython-release-tracker/pull/8) as well, no?

Yep! I was going to re-run the script to pull that change in, but a manual change to it also works 🙂

File: action.yml [hunk 1]
```diff
@@ -33,7 +33,7 @@ runs:
         repository: woodruffw/cpython-release-tracker
         persist-credentials: false
         path: cpython-release-tracker
-        ref: 9eb74dae56e55433ee03c7283dc013645d5d8c4e
+        ref: 331017869cd86699aaae8f4fadf003e4e36b1c2e
```
Comment (woodruffw) [inline action.yml:36]:
```suggestion
        ref: 124a4005d3822cc21f16396e81bf08af3e858561
```

**Comment by woodruffw** [comment]
Yep! Looks like we're green again here, so CCing @kommendorkapten et al for approval 🙂

**Review by haydentherapper — APPROVED** [review]

**Pull request merged**